viewer/src/data/network/client/ModelSectorService.ts

```diff
@@ -0,0 +1,7 @@
+/*!
+ * Copyright 2020 Cognite AS
+ */
+
+export interface ModelSectorService {
+  fetchModelSector(modelId: number, file: string): Promise<Uint8Array>;
```

Comment:
Is this implemented yet?

Reply:
I did add a RevealClient.ts which uses this right?

Reply:
Yes, I saw the usage, but not the implementation. Is that added yet?

Reply:
Perhaps you forgot to add a file?

Reply:
No, I forgot, I added it last night but didn't push it yet xD I'll add it once I have solved my current brain puzzle